examples/odata-cli/src/activities.ts

```diff
@@ -20,6 +20,7 @@ export async function useCatalogAndFetchSomeMetadata(
     try {
         const atoSettings = await provider.getAtoInfo();
         if (!atoSettings || Object.keys(atoSettings).length === 0) {
+            // eslint-disable-next-line no-console
             console.warn('ATO setting is empty!');
```

Comment:
Should be using the logging module?

Reply:
This is an example. So console should be ok?

Reply:
1 vote to leave the `console.warn` as it is. 
It's an example, let's keep it simple and only use the code that this is example for